pom.xml

```diff
@@ -4,7 +4,7 @@
     <artifactId>open-dis</artifactId>
     <packaging>jar</packaging>
     <name>Open-DIS</name>
-    <version>5.1-SNAPSHOT</version>
+    <version>5.1.1-SNAPSHOT</version>
```

Comment:
This line can stay 5.1-SNAPSHOT until we cut a release.

Reply:
Can you revert this line.

Reply:
Done